src/judges/codeforces/getContests.js

```diff
@@ -1,46 +1,39 @@
 // @flow
-import request from 'request';
+import request from 'request-promise-native';
 import type { Contest } from '../../types';
 
-export function getContests(): Promise<Array<Contest>> {
-  return new Promise((resolve, reject) => {
-    // making request to the codeforces api
-    request('http://codeforces.com/api/contest.list', (error: Object, response: Object, body: string) => {
-      if (response && response.statusCode == 200) {
-        let contests: Array<Contest> = [];
+export async function getContests(): Promise<Array<Contest>> {
+  const body = await request('http://codeforces.com/api/contest.list');
+  const contests: Array<Contest> = [];
 
-        let data: Object = JSON.parse(body).result;
-        for (let index: string in data) {
-          let info: Object = data[index];
+  const data: Object = JSON.parse(body).result;
```

Comment:
I don't like `data`. Maybe `cfResponse`? Or even `codeforcesResponse`

Reply:
I don't know. But `data` is way too generic.

Reply:
There's no need to repeat this is about codeforces. It is on the codeforces file and it is local. I don't mind `data`, sounds perfect to me, really, describes exactly what it means to the context that matters to it.

Reply:
There's no need to repeat this is about codeforces. It is on the codeforces file and it is local. I don't mind `data`, sounds perfect to me, really, describes exactly what it means to the context that matters to it.